WcaOnRails/app/controllers/admin_controller.rb

```diff
@@ -26,4 +26,43 @@ def do_merge_people
     end
     render 'merge_people'
   end
+
+  def edit_person
+    # This is necessary because the simple form needs some real active record when using user_ids picker.
+    # That's only to pass the appropriate attributes.
+    @person = Person.new
+  end
+
+  def update_person
+    @person = Person.current.find_by!(wca_id: params[:person][:wca_id])
+    person_params = params.require(:person).permit(:name, :countryId, :gender, :dob)
+
+    case params[:commit]
+    when "Fix"
+      if @person.update_attributes(person_params)
+        flash.now[:success] = "Successfully fixed #{@person.name}."
```

Comment:
In the event of success, we should redirect back to `/admin/edit_person`. I just tried copying the url I was at after doing a fix, and I ended up with `/admin/update_person`, which is weird when I tried to send the link to you =)